### PR TITLE
Removed the drag and carry command while jailed

### PR DESCRIPTION
**Pull request by DatBoiWeep**

### CODE REVIEW

File: resources/usa_drag/client.lua [hunk 1]
```diff
@@ -10,19 +10,21 @@ local isCarried = false
...
 	end
-	if not isDragging and not isDragged and not isCarrying and not isCarried then
+	if not isDragging and not isDragged and not isCarrying and not isCarried and not jailed then
 		TriggerEvent("usa:getClosestPlayer", 1.65, function(player)
     		if player then
 	        	local closestPed = GetPlayerPed(GetPlayerFromServerId(player.id))
 				if player.id ~= 0 and not IsPedInAnyVehicle(PlayerPedId()) and IsEntityVisible(closestPed) then
 					TriggerServerEvent('drag:sendDragPlayer', player.id)
-					--print('sent??')
+					print('sent??')
```
Comment (minipunch) [inline resources/usa_drag/client.lua]: plz remove debug prints when pushing live

File: resources/usa_drag/client.lua [hunk 2]
```diff
@@ -135,11 +137,16 @@ AddEventHandler('drag:toggleCarryAction', function(toggleOn, _source)
 	ClearPedTasksImmediately(PlayerPedId())
 end)
 
+RegisterNetEvent("death:toggleJailed")
+AddEventHandler("death:toggleJailed", function(toggle)
```
Comment (minipunch) [inline resources/usa_drag/client.lua:141]: If we're going to do it this way, can you at least rename the event from `death:toggleJailed` to something more generic so it makes more sense when used not in the `usa_death` script

Something like `usa:toggleJailedStatus`

**Review by minipunch — CHANGES_REQUESTED** [review]
see comments

File: resources/usa_drag/client.lua (hunk 1 above)
Comment (DatBoiWeep) [reply resources/usa_drag/client.lua]: Removed :)

File: resources/usa_drag/client.lua (hunk 2 above)
Comment (DatBoiWeep) [reply resources/usa_drag/client.lua:141]: On it boss

**Review by minipunch — APPROVED** [review]

**Comment by minipunch** [comment]
hoping you ctrl + shift + f'd the whole repo to find all the scripts to rename all the events lul

**Pull request merged**